en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md

```diff
@@ -450,41 +458,34 @@ keep your bearings.
 ## Using Pandoc to convert Markdown to an MS Word document
 
 We are now ready to typeset! Open your terminal window, use `$ pwd` and
-`$ cd` to navigate to the correct folder for your project. Once you are
+`$ cd DIRECTORY-NAME` to navigate to the correct folder for your project. Once you are
 there, type `$ ls` in the terminal to list the files. If you see your
 .md file and your images, you are in the right place. To convert .md
 into .docx type:
 
 ```
-$ pandoc -o main.docx main.md
+$ pandoc main.md -o main.docx
```

Comment:
While the new command configuration is valid, the [canonical Pandoc documentation](https://pandoc.org/MANUAL.html#using-pandoc) still uses the formulation `pandoc [options] [input-file]...` I hesitate to change this tutorial in order to pull it farther out of sync with Pandoc's actual documentation, and thus cause confusion for users trying to square what we say versus what's in the docs.

Reply:
Good point, though in teaching this tutorial I've now began to advocate for `pandoc [options] input-file -o [output-file]`. The problem with -o is that it must be followed by the output file, making it so that the output path changes position arbitrarily depending on the order of operations. Putting it always as the last argument after the input-file is easier to read and remember. The audience for this tutorial is unlikely to be confused by a more logical order of operations. The majority encounter Pandoc for the first time here, and the few that are familiar with the canonical documentation understand that Shell commands allow for composable syntax. 

In short, I think we should go with `pandoc [options] input-file -o [output-file]` wherever possible because it speaks to "sustainability." I will be happy to revert it if we receive feedback otherwise.

Reply:
I'm convinced! 👍